observability/otel/otel_rn/otel-rn-3-2-1.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="otel-rn-3-2-1"]
+= Release notes for {OTELName} 3.2.1
+:context: otel-rn-3-2-1
+
+toc::[]
+
+include::modules/otel-product-overview.adoc[leveloffset=+1]
+
+The {OTELName} is provided through the {OTELOperator}.
+
+[id="otel-rn_3-2-1_cves"]
+== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-25062/[CVE-2024-25062]
```

Comment:
@rubenvp8510 how do we now that we were affected by this CVE?  

it is not reflected in the current health index https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-collector-rhel8/613b50f3a052bb39f2c5f31f?architecture=amd64&image=66604eb66955aa0ea39f7586&container-tabs=overview

Reply:
That is the thing, it seems like is on the base image, and  I had to rebuild due a warning regarding of that CVE. not sure why doesn't appear in the health index.

Reply:
We discussed it during today's team call and Matt suggested that we include this CVE.